akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolConductor.scala

```diff
@@ -137,6 +146,9 @@ private object PoolConductor {
               slotStates(slotIx) = slotStateAfterRequestCompleted(slotStates(slotIx))
             case SlotEvent.Disconnected(slotIx, failed) ⇒
               slotStates(slotIx) = slotStateAfterDisconnect(slotStates(slotIx), failed)
+              reconnectIfNeeded()
+            case SlotEvent.ConnectedEagerly(slotIx) ⇒
+              slotStates(slotIx) = Idle
```

Comment:
Doesn't this prevent requests from dispatch on this slot between sending `SlotShouldConnectCommand` and receiving  `ConnectedEagerly`? Previously dispatch of the first requests instantly set the slot state to `Loaded`/`Busy`. Maybe it should become `Idle` right away.